### PR TITLE
328 complete canada eligibility checks details page caroline

### LINKED ISSUE #328: Complete Canada Eligibility Checks details page (Caroline)

### PR DESCRIPTION
**Pull request by cazcam34**
Initial round of release notes for visa eligibility checks... English writing isn't a strong suit, so please edit away!!! I've tried to keep it fairly broad and not Canada specific. I think it could be a good way to introduce this new visa check format and explain it to everyone, particularly before we look to rolling out across destinations. Anyways not sure if it's too wordy or not the right tone for the release notes, so very open to feedback!! I am yet to do my video, so I'll get that done tomorrow (Tuesday) before our meeting and I can make changes based on your feedback.

### CODE REVIEW

File: docs/v210/visa_eligibility_details.md [hunk 1]
```diff
@@ -6,10 +6,62 @@ sass:
...
+  <img src="./../assets/images/v210/CanadaVisaCheckSectionOne.png" alt="Enhanced Other Languages Search" class="card-image">
+</div>
+
+# Section 2 - Job Specific Assessment Questions
+
+The second section of the visa eligibility check collects data that will change depending on the particular job 
+opportunity. Data collected is job specific, such as relevant candidate experience for the role and 
+candidate's interest in the role etc. When adding a job to the candidate's visa eligibility check it is automated to 
+only show the candidate's job opportunities as selections to streamline the process and connect the associated job's 
+details to the visa check.
```
Comment (camerojo) [inline docs/v210/visa_eligibility_details.md]: I think that this sentence is a bit confusing. How about " When adding a job to the candidate's visa eligibility check only that candidate's job opportunities are presented for selection."

**Review by camerojo — DISMISSED** [review]
Looks good. Approved. Just one comment about a sentence I found a bit confusing.

File: docs/v210/visa_eligibility_details.md (hunk 1 above)
Comment (cazcam34) [reply docs/v210/visa_eligibility_details.md]: Eep yes that sentence got lost somewhere! Thanks I've changed as you suggested.

**Review by camerojo — APPROVED** [review]
Approved
